components/clientComponents/forms/Review/Review.tsx

```diff
@@ -61,7 +61,8 @@ function formatElementValues(element: ReviewElement) {
   return String(element.values);
 }
 
-function getReviewItemElements(
+// TODO: move into a helper file?
+export function getReviewItemElements(
```

Comment:
Just looking at the name of this function 

What's a ReviewItem ? --- trying to remember why it's called that.